src/cmd/easee_internal_test.go

```diff
@@ -369,6 +369,314 @@ func TestEaseeEdgeApp(t *testing.T) { //nolint:paralleltest
 					},
 				},
 			},
+			{
+				Name: "Inclusion report updated",
+				Setup: serviceSetup(
+					testContainer,
+					"configured",
+					func(client *mocks.APIClient) {
+						client.On("ChargerConfig", "XX12345").Return(&model.ChargerConfig{
+							DetectedPowerGridType: model.GridTypeTN3Phase,
+							PhaseMode:             1,
+						}, nil)
+						client.On("ChargerSiteInfo", "XX12345").Return(&model.ChargerSiteInfo{
+							RatedCurrent: 32,
+						}, nil)
+						client.On("Ping").Return(nil)
+					},
+					signalRSetup(test.DefaultSignalRAddr, func(s *test.SignalRServer) {
+						s.MockObservations(0, []signalr.Observation{
+							{
+								ChargerID: test.ChargerID,
+								DataType:  model.ObservationDataTypeInteger,
+								ID:        model.ChargerOPState,
+								Value:     strconv.Itoa(int(model.ChargerStateAwaitingStart)),
+							},
+							{
+								ChargerID: test.ChargerID,
+								DataType:  model.ObservationDataTypeDouble,
+								ID:        model.MaxChargerCurrent,
+								Value:     "32",
+							},
+							{
+								ChargerID: test.ChargerID,
+								DataType:  model.ObservationDataTypeInteger,
+								ID:        model.DetectedPowerGridType,
+								Value:     "1",
+							},
+							{
+								ChargerID: test.ChargerID,
+								DataType:  model.ObservationDataTypeInteger,
+								ID:        model.PhaseMode,
+								Value:     "2",
+							},
+						})
+					})),
+				TearDown: []suite.Callback{tearDown("configured"), testContainer.TearDown()},
+				Nodes: []*suite.Node{
+					suite.SleepNode(500 * time.Millisecond),
+					{
+						InitCallbacks: []suite.Callback{waitForRunning()},
+						Command:       suite.StringMessage("pt:j1/mt:cmd/rt:ad/rn:easee/ad:1", "cmd.thing.get_inclusion_report", "easee", "1"),
+						Expectations: []*suite.Expectation{
+							suite.ExpectObject("pt:j1/mt:evt/rt:ad/rn:easee/ad:1", "evt.thing.inclusion_report", "easee", inclusionReportValueUpdate("TN", []string{"NL1", "NL2", "NL3", "NL1L2L3"}, 3, true, chargepointAllPropsSrv)),
```

Comment:
I think we can go with custom assertions here to ensure we check only those values we're actually interested about. I'd go with our own `Expect` function, but it's just an example. If you have questions, you can always poke me and discuss it.